### DISCUSSION TITLE
Does OCRmyPDF transfer form fields when you use it with optimize?

### DISCUSSION THREAD

**Discussion by wobeng** [ocrmypdf/OCRmyPDF#1280] [Q&A]
Hello :)

Does OCRmyPDF transfer form fields when you use it with optimize?

**Comment by jbarlow83** [comment]
If you use `--output-type pdf` and something like `--skip-text` then form fields (populated or otherwise) will be passed through.

If you use `--output-type pdfa` (the default) form fields will be "burned in" and deactivated (the answers will be permanently written in the document, and they will no longer be interactive elements). Likewise for `--force-ocr`.

Generally speaking documents that need OCR don't have form fields.
